docs/contributing/code_contribution_guide.rst

```diff
@@ -90,6 +90,19 @@ branch which is your feature branch pushed at your fork.
 .. note::
     Do not push to ``master`` or ``devel`` branch on your fork or upstream.
 
+Best Practices when creating Pull Request
```

Comment:
Please capitalize the "W" in When and "C" in creating.

Reply:
fixed